.gitlab/ci_cd.yaml

```diff
@@ -108,7 +167,7 @@ variables:
       fi
 
     # Run data quality tests
-    - tb test run -v -c 4
+    - tb --semver $VERSION test run -v -c 4
```

Comment:
Running explicitly in order not to be dependent on was promote or not?

Reply:
yes, I'm doing the same with fixture tests, I have pending to update the script